web-client/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 use std::{cell::RefCell, collections::HashMap, rc::Rc, str::FromStr};
 
+use address::Address;
```

Comment:
```suggestion
use crate::address::Address;
```
(and move it to the list where all `use crate::` are)